### ISSUE TITLE
Motion mask "copy" bug when multiple maks are created

### ISSUE THREAD

**Issue by FlexxFR** [blakeblackshear/frigate#1798]
**Describe the bug**
Only a small issue but probably easy to fix.
When using the mask creator and adding a second mask the "copy" function does not work properly
It captures the "REMOVE" after the first line.

motion:
  mask:
    - 1024,172,1024,0,0,0,0,144,528,119,533,97,700,133,703,105**REMOVE**
    - 848,576,1024,576,1024,137

Of course easy to correct during copy/paste.

**Version of frigate**
Current stable release

**Comment by spikeygg** [comment]
I just noticed this bug too when trying to manage masks. When clicking the "Copy" button after multiple masks have been created, the clipboard receives some boiled down broken mixture of them. I came here to open a ticket but @FlexxFR beat me to it. 🥇 

When I click copy here:
![image](https://github.com/blakeblackshear/frigate/assets/1015891/4dc769b8-2657-410d-92b8-cbda884f1c8c)

I get this into the clipboard:
```
  objects:
    filters:
      person:
        mask: 1984,521,2121,521,3698,1536,3957,1327
```

What the duce?!
